AppDashboard/templates/logs/main.html

```diff
@@ -2,14 +2,17 @@
 <!--Body content-->
 <div id="content" class="clearfix">
     <div class="contentwrapper"><!--Content wrapper-->
-
+        {% set app_page_title="Log Viewer" %}
```

Comment:
The title of the page? Its used as the panel title or the app_page title if app_page sets standalone_app_page to True. Rather than doing jQuery calls to grab the inner HTML of the heading, I decided to set a variable that both of those templates use. The reason for the if statement is simply that I can't access app_page_title in the app_page template where I would need to.